code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1684,6 +1684,13 @@ var/global/list/obj/item/device/pda/PDAs = list()
 					if (mode == 1)
 						note = replacetext(n, "\n", "<BR>")
```

Comment:
Because that's setting the actual note that's showing on the PDA when you open it up, complete with new lines.